README.md

```diff
@@ -122,25 +122,29 @@ additional metrics!
 [Heapster](https://github.com/kubernetes/heapster) is a project which fetches
 metrics (such as CPU and memory utilization) from the Kubernetes API server and
 nodes and sends them to various time-series backends such as InfluxDB or Google
-Cloud Monitoring.
+Cloud Monitoring. Its most important function right now is implementing certain
+metric APIs that Kubernetes components like the horizontal pod auto-scaler
+query to make decisions.
 
 While Heapster's focus is on forwarding metrics already generated by
 Kubernetes, kube-state-metrics is focused on generating completely new metrics
 from Kubernetes' object state (e.g. metrics based on deployments, replica sets,
 etc.). The reason not to extend Heapster with kube-state-metrics' abilities is
 because the concerns are fundamentally different - while Heapster only needs to
-fetch, format and forward metrics that already exist, kube-state-metrics holds
-an entire snapshot of Kubernetes state in memory and continuously generates new
-metrics based off of it but has no responsibility for exporting its metrics
-anywhere.
+fetch, format and forward metrics that already exist, in particular from Kubernetes
+components and write them into sinks. The sinks are the actual monitoring systems.
+kube-state-metrics holds an entire snapshot of Kubernetes state in memory and
+continuously generates new metrics based off of it but has no responsibility for
+exporting its metrics anywhere.
 
 In other words, kube-state-metrics itself is designed to be another source for
 Heapster (although this is not currently the case).
 
-Additionally, some monitoring systems such as Prometheus do not use Heapster
-for metric collection at all and instead implement their own. Having
-kube-state-metrics as a separate project enables access to these metrics from
-those monitoring systems.
+Additionally, some monitoring systems such as Prometheus do not use Heapster for
+metric collection at all and instead implement their own, but the Prometheus can
```

Comment:
but Prometheus can scrape metrics from heapster itself to alert on heapster's health